src/api/store/community.py

```diff
@@ -340,88 +364,64 @@ def create_community(self,context: Context, args) -> (dict, MassEnergizeAPIError
       #TODO: make a copy of the images instead, then in the home page, you wont have to create new files everytime
       if homePage:
         homePage.pk = None 
-        homePage.title = f"Welcome to Massenergize, {new_community.name}!"
-        homePage.community = new_community
+        homePage.title = f"Welcome to Massenergize, {community.name}!"
+        homePage.community = community
         homePage.is_template = False
         homePage.save()
         homePage.images.set(images)
+
+      #now create all the pages
+      if not _clone_page_settings(AboutUsPageSettings, f"About {community.name}", community):
+        raise("Failed to clone settings for AboutUs page")
+      if not _clone_page_settings(ActionsPageSettings, f"Actions for {community.name}", community):
+        raise("Failed to clone settings for Actions page")
+      if not _clone_page_settings(ContactUsPageSettings, f"Contact Us - {community.name}", community):
+        raise("Failed to clone settings for ContactUs page")
+      if not _clone_page_settings(DonatePageSettings, f"Take Actions - {community.name}", community):
+        raise("Failed to clone settings for Donate page")
+      if not _clone_page_settings(ImpactPageSettings, f"See our Impact - {community.name}", community):
+        raise("Failed to clone settings for Impact page")
+      if not _clone_page_settings(TeamsPageSettings, f"Teams in this community", community):
+        raise("Failed to clone settings for Teams page")
+      if not _clone_page_settings(VendorsPageSettings, f"Service Providers", community):
+        raise("Failed to clone settings for Vendors page")
+      if not _clone_page_settings(EventsPageSettings, f"Events and Campaigns", community):
+        raise("Failed to clone settings for Events page")
+      if not _clone_page_settings(TestimonialsPageSettings, f"Testimonials", community):
+        raise("Failed to clone settings for Testimonials page")
     
-      impactPage = ImpactPageSettings.objects.filter(is_template=True).first()
-      if impactPage:
-        impactPage.pk = None 
-        impactPage.title = f"See our Impact - {new_community.name}"
-        impactPage.community = new_community
-        impactPage.is_template = False
-        impactPage.save()
-
-      # by adding TeamsPageSettings - since this doesn't exist for all communities, will it cause a problem? 
-      # Create it when TeamsPageSettings in admin portal
-      teamsPage = TeamsPageSettings.objects.filter(is_template=True).first()
-      if teamsPage:
-        teamsPage.pk = None 
-        teamsPage.title = f"Teams in this community"
-        teamsPage.community = new_community
-        teamsPage.is_template = False
-        teamsPage.save()
-    
-      admin_group_name  = f"{new_community.name}-{new_community.subdomain}-Admin-Group"
-      comm_admin: CommunityAdminGroup = CommunityAdminGroup.objects.create(name=admin_group_name, community=new_community)
+      print("Create admin group")
```

Comment:
let's remove.  If you need it temporarily just to for debugging or testing i think put a // TODO comment to remove afterwards so you can trace back to it.

Reply:
Done